nixos/modules/services/security/munge.nix

```diff
@@ -43,25 +43,22 @@ in
 
     users.groups.munge = {};
 
+    systemd.tmpfiles.rules = [
+      "d /var/lib/munge 0711 munge munge - -"
+      "d /run/munge - munge munge - -"
+      "e '${cfg.password} 0400 - - - -"
```

Comment:
I am not familiar yet with the new tmpfiles feature but the man page of 'tmpfiles.d' states for `e`:
> Similar to d, but the directory will not be created if it does not exist

`cfg.password` is a file not a directory. Is this the right way to to do it?

Reply:
@markuskowa I guess I thought that would apply to files as well, but reading the docs again it does not state that. I've replaced that commit with this: https://github.com/NixOS/nixpkgs/pull/56265/commits/422fd65c0872732d3d8c3b4b32de10a73e339e80

How does that look?

Reply:
@markuskowa ping

Reply:
The slurm test runs. I think this solution is OK now.